tensorflow_cloud/machine_config.py

```diff
@@ -151,12 +151,6 @@ def validate(self):
         accelerator_type=AcceleratorType.NVIDIA_TESLA_V100,
         accelerator_count=4,
     ),
-    "V100_8X": MachineConfig(
```

Comment:
Removed this as 8 V100s cannot be created. It is above the allowed limit of 4 for Cloud ML engine